integrations/streamer_test.go

```diff
@@ -2534,7 +2534,10 @@ func TestStream_AlertAlerta(t *testing.T) {
 			if exp := "cpu"; pd.Resource != exp {
 				t.Errorf("unexpected resource got %s exp %s", pd.Resource, exp)
 			}
-			if exp := "production"; pd.Environment != exp {
+			if exp := "TestStream_Alert"; pd.Event != exp {
```

Comment:
based on your event template `event: {{ .TaskName }}` the expected value should be `exp := "event: TestStream_Alert"`